api/agent_access_tokens.go

```diff
@@ -169,3 +169,71 @@ type AgentAccessTokenRequest struct {
 	TokenAlias string                 `json:"tokenAlias,omitempty"`
 	Props      *AgentAccessTokenProps `json:"props,omitempty"`
 }
+
+func (svc *AgentAccessTokensService) Search(response interface{}, filters SearchFilter) error {
+	return svc.client.RequestEncoderDecoder("POST", apiV2AgentInfoSearch, filters, response)
+}
+
+type AgentInfoResponse struct {
+	Data   []AgentInfo  `json:"data"`
+	Paging V2Pagination `json:"paging"`
+}
+
+// Fulfill Pageable interface (look at api/v2.go)
+func (r AgentInfoResponse) PageInfo() *V2Pagination {
+	return &r.Paging
+}
+func (r *AgentInfoResponse) ResetPaging() {
+	r.Paging = V2Pagination{}
+}
+
+type AgentInfo struct {
+	AgentVersion string    `json:"agentVersion"`
+	CreatedTime  time.Time `json:"createdTime"`
+	Hostname     string    `json:"hostname"`
+	IpAddr       string    `json:"ipAddr"`
+	LastUpdate   string    `json:"lastUpdate"`
```

Comment:
We need to change this to:
```suggestion
	LastUpdate   time.Time    `json:"lastUpdate"`
```
But we need to wait for this change to be released: https://github.com/lacework/services/pull/8095

Jira: https://lacework.atlassian.net/browse/RAIN-34237